### ISSUE TITLE
mbs_bin_dir should be configurable in the GUI

### ISSUE THREAD

**Issue by ninneman** [ninneman/mbgui#7]
The variable mbs_bin_dir, used to tell mbgui where to find the MB-System binaries, should be configurable in the GUI. This raises the issue of how to make configuration state persist across separate mbgui runs.

**Comment by ninneman** [comment]
It can be a variable that's set each time, but that's user-hostile. Once a user installs MB-System, they're not likely to change where its binaries live. Asking them to give the same data over and over again is sloppy. It would do as a stop-gap though.

**Comment by kalcan83** [comment]
this is obviously lower priority... but what if there was a pop-up window that would edit that variable if/when needed... - like an advanced option

**Comment by ninneman** [comment]
Yeah, that's fine. The issue is how to preserve what the user sets across subsequent runs of mbgui. What I'll do for starters is have mbgui write its config data to a file.
